cmake/dca_compiler_options.cmake

```diff
@@ -41,6 +41,11 @@ if (DCA_HAVE_CUDA)
       -Xcompiler -Wno-unused-parameter
       -Xcompiler -Wno-switch
       -Xcompiler ${DCA_THREADING_FLAGS})
+    if (CMAKE_BUILD_TYPE MATCHES "Debug")
+      list(APPEND CUDA_NVCC_FALGS
```

Comment:
Typo. Does the code still work if the user passed the same flag to the cached CUDA_NVCC_FLAGS variable?